xlsx.cabal

```diff
@@ -36,7 +36,8 @@ Flag microlens
 
 Library
   Hs-source-dirs:    src
-  ghc-options:       -Wall
+  profiling: False
+  ghc-options:       -Wall -O2
```

Comment:
I asked a colleague, There were some issues in getting xlsx to use 02 in ghci for benchmarking, this was one of many attempts to correct that. I'm removing it since for our internal code we can use nix to modify this anyway.

Reply:
Thanks